changelog/unreleased/issue-3117

```diff
@@ -0,0 +1,4 @@
+Enhancement: json output for command forget extended
```

Comment:
What about `Enhancement: include snapshot id in reason field of forget JSON output` or something similar?

Reply:
description as suggested